ruby-3.0.2-circleci/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ruby:3.0.2-buster
```

Comment:
previously we made these from the production base image to keep things as consistent as possible. any reason not to? https://github.com/ParkWhiz/rails-base/blob/master/ruby-2.6.3-circleci/Dockerfile

Reply:
nope, no real reason not to. i figured we used to do that because installing ruby and setting stuff up like that was a lot of duplication. but now that we're on the main ruby container, there's next to no duplication.

i'll change it to use the base container, though